text/0308-cli-advisories.md

```diff
@@ -0,0 +1,263 @@
+# CLI advisories
+
+* **Original Author(s)**: [@otaviomacedo](https://github.com/otaviomacedo)
+* **Tracking Issue**: [#308](https://github.com/aws/aws-cdk-rfcs/issues/308)
+* **API Bar Raiser**: @{BAR_RAISER_USER}
+
+A new CLI feature to notify customers about urgent and important issues that
+require their attention.
+
+## Working backwards
+
+Starting on version x.y.z of the CDK CLI, customers will be notified, on every
+command, about security vulnerabilities, regressions and usage of unsupported
+versions:
+
+```
+$ cdk deploy
+
+... # Normal output of the command
+
+ADVISORIES
+
+16603   Toggling off auto_delete_objects for Bucket empties the bucket
+
+        Summary: If a stack is deployed with an S3 bucket with
+                 auto_delete_objects=True, and then re-deployed with 
+                 auto_delete_objects=False, all the objects in the bucket 
+                 will be deleted.
+                 
+        Affected versions: <1.126.0.
+
+        More information at: https://github.com/aws/aws-cdk/issues/16603
+
+
+17061   Error when building EKS cluster with monocdk import
+
+        Summary: When using monocdk/aws-eks to build a stack containing
+                 an EKS cluster, error is thrown about missing 
+                 lambda-layer-node-proxy-agent/layer/package.json.
+         
+        Affected versions: >=1.126.0 <=1.130.0.
+
+        More information at: https://github.com/aws/aws-cdk/issues/17061 
+
+If you don’t want to see an advisory anymore, use "cdk acknowledge ID". For example, "cdk acknowledge 16603".
+```
+
+By acknowledging a particular advisory, it won’t show anymore in subsequent calls:
+
+```
+$ cdk acknowledge 16603
+
+ADVISORIES
+
+17061   Error when building EKS cluster with monocdk import
+
+        Summary: When using monocdk/aws-eks to build a stack containing
+                 an EKS cluster, error is thrown about missing 
+                 lambda-layer-node-proxy-agent/layer/package.json.
+         
+        Affected versions: >=1.126.0 <=1.130.0.
+
+        More information at: https://github.com/aws/aws-cdk/issues/17061 
+
+If you don’t want to see an advisory anymore, use "cdk acknowledge ID". For example, "cdk acknowledge 17061".
+```
+
+You can suppress all warnings per individual execution:
+
+```
+$ cdk deploy --no-advisories
+```
+
+And you can disable all advisories indefinitely by adding this entry to
+`~/.cdk.json`:
+
+```
+"supressAllAdvisories": true
```

Comment:
This should just be `advisories: false` (which is exactly the same as `--no-advisories` in yargs).

Reply:
Done.